src/EditorFeatures/CSharpTest/CSharpEditorServicesTest.csproj

```diff
@@ -205,6 +205,7 @@
     <Compile Include="Diagnostics\UseImplicitOrExplicitType\UseExplicitTypeTests_FixAllTests.cs" />
     <Compile Include="Diagnostics\UseImplicitOrExplicitType\UseImplicitTypeTests_FixAllTests.cs" />
     <Compile Include="Diagnostics\UseImplicitOrExplicitType\UseImplicitTypeTests.cs" />
+    <Compile Include="DocumentationComments\CodeFixes\AddDocCommentParamNodeCodeFixProviderTests.cs" />
```

Comment:
The other one is more generic.  I can rename it, but it would probably need some modifications if, say, you tried to use it to add an exception node.